js/data/buffer/fill_vertex_buffer.js

```diff
@@ -1,25 +1,24 @@
 'use strict';
 
 var util = require('../../util/util');
-var Buffer = require('./buffer');
-
-module.exports = FillVertexBuffer;
-
-function FillVertexBuffer(buffer) {
-    Buffer.call(this, buffer);
+var Buffer2 = require('../buffer2');
+
+function FillVertexBuffer(options) {
+    Buffer2.call(this, options || {
+        type: Buffer2.BufferType.VERTEX,
+        attributes: {
```

Comment:
The name for the element buffers doesn't actually matter... I was using `vertices`. The entires in an element buffer aren't "positions" -- they are indices of elements in the vertex buffer.